src/components/svgs/stations/qingdao-metro-basic.tsx

```diff
@@ -0,0 +1,217 @@
+
+    const onPointerDown = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerDown(id, e),
+        [id, handlePointerDown]
+    );
+    const onPointerMove = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerMove(id, e),
+        [id, handlePointerMove]
+    );
+    const onPointerUp = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerUp(id, e),
+        [id, handlePointerUp]
+    );
+
+    const textX = nameOffsetX === 'left' ? -7.5 : nameOffsetX === 'right' ? 7.5 : 0;
+    const textY =
+        (names[NAME_DY[nameOffsetY].namesPos].split('\\').length * NAME_DY_QD_BASIC[nameOffsetY].lineHeight +
+            NAME_DY_QD_BASIC[nameOffsetY].offset) *
+        NAME_DY_QD_BASIC[nameOffsetY].polarity;
+    const textAnchor = nameOffsetX === 'left' ? 'end' : nameOffsetX === 'right' ? 'start' : 'middle';
+
+    return (
+        <g id={id} transform={`translate(${x}, ${y})`}>
+            <circle
+                id={`stn_core_${id}`}
+                r={3}
+                stroke={color[2]}
+                strokeWidth="0.67"
+                fill="white"
+                onPointerDown={onPointerDown}
+                onPointerMove={onPointerMove}
+                onPointerUp={onPointerUp}
+                style={{ cursor: 'move' }}
+            />
+            <g transform={`translate(${textX}, ${textY})`} textAnchor={textAnchor}>
+                <MultilineText
+                    text={names[0].split('\\')}
+                    fontSize={10}
+                    lineHeight={10}
+                    grow="up"
+                    baseOffset={1}
+                    className="rmp-name__zh"
+                />
+                <MultilineText
+                    text={names[1].split('\\')}
+                    fontSize={5.5}
+                    lineHeight={5.5}
+                    grow="down"
+                    baseOffset={1.5}
+                    className="rmp-name__en"
+                />
+            </g>
+        </g>
+    );
+};
+
+/**
+ * qingdaoMetroBasicStation specific props.
+ */
+export interface QingdaoMetroBasicStationAttributes extends StationAttributes, AttributesWithColor {
+    nameOffsetX: NameOffsetX;
+    nameOffsetY: NameOffsetY;
+}
+
+const defaultQingdaoMetroBasicStationAttributes: QingdaoMetroBasicStationAttributes = {
+    ...defaultStationAttributes,
+    color: [CityCode.Qingdao, 'qd1', '#eaaa00', MonoColour.white],
+    nameOffsetX: 'right',
+    nameOffsetY: 'top',
+};
+
+const qingdaoMetroBasicAttrsComponent = (props: AttrsProps<QingdaoMetroBasicStationAttributes>) => {
+    const { id, attrs, handleAttrsUpdate } = props;
+    const { t } = useTranslation();
+
+    const fields: RmgFieldsField[] = [
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameZh'),
+            value: attrs.names[0].replaceAll('\\', '\n') ?? defaultQingdaoMetroBasicStationAttributes.names[0],
```

Comment:
No need for `.replaceAll('\\', '\n')` any more. It's only used for backwards compatibility for older stations that use `\\` as a line breaker.